FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SetUp/content.txt

```diff
@@ -1,7 +1,9 @@
 !|dbfit.TeradataTest|
 
+|Connect Using File|!-TestDbConnectionDbFitTeradata.properties-!|
```

Comment:
FitNesse presents the word like this: -

![image](https://user-images.githubusercontent.com/1895701/27725151-f8b2db2a-5d6c-11e7-82ea-947ada691df5.png)

The URL has the suffix `?edit&nonExistant=true`.